### PR TITLE
ISSUE 28305 treat mariaDB as an independent database type

### PR DESCRIPTION
**Pull request by ben-enfuse-io**
Signed-off-by: Mayra Quiroga <mq@enfuse.io>

### CODE REVIEW

File: spring-jdbc/src/main/java/org/springframework/jdbc/support/JdbcUtils.java [hunk 1]
```diff
@@ -455,9 +455,6 @@ public static String commonDatabaseName(@Nullable String source) {
 		if (source != null && source.startsWith("DB2")) {
 			name = "DB2";
 		}
-		else if ("MariaDB".equals(source)) {
```
Comment (vasaau) [inline spring-jdbc/src/main/java/org/springframework/jdbc/support/JdbcUtils.java:458]: why this has been removed?

File: spring-jdbc/src/main/java/org/springframework/jdbc/support/JdbcUtils.java [hunk 2]
```diff
@@ -455,9 +455,6 @@ public static String commonDatabaseName(@Nullable String source) {
...
-			name = "MySQL";
-		}
 		else if ("Sybase SQL Server".equals(source) ||
```
Comment (vasaau) [inline spring-jdbc/src/main/java/org/springframework/jdbc/support/JdbcUtils.java:458]:
```suggestion
		else if ('Sybase SQL Server'.equals(source) ||
```

**Comment by tyler2cr** [comment]
Resolves https://github.com/spring-projects/spring-framework/issues/28305

**Comment by tyler2cr** [comment]
Resolves https://github.com/spring-cloud/spring-cloud-task/issues/833

**Pull request closed without merging**